src/applications/find-forms/api/index.js

```diff
@@ -26,11 +26,16 @@ export const fetchFormsApi = async (query, options = {}) => {
 
   const forms = response?.data;
   const onlyValidForms = forms?.filter(
-    form => form.attributes?.validPDF || form.attributes?.validPdf,
+    form =>
+      (form.attributes?.validPDF || form.attributes?.validPdf) &&
+      form.attributes?.deletedAt === null,
```

Comment:
Can `deletedAt` be `undefined`?

Reply:
It did not appear to be when scanning responses; however, it's probably good to check for it just in case.